### PR TITLE
Add state as a routing method in the client model

### PR DESCRIPTION
**Pull request by embarnard**

### CODE REVIEW

File: spec/features/web_intake/routing_spec.rb [hunk 1]
```diff
@@ -84,6 +86,39 @@
...
+    click_on "File taxes with help"
+
+    expect(page).to have_text "File with the help of a tax expert!"
+    click_on "Continue"
+
+    expect(page).to have_text "What years do you need to file for?"
+    check "2020"
+    click_on "Continue"
+
+    expect(Intake.last.source).to eq nil
```
Comment (alaroia-cfa) [inline spec/features/web_intake/routing_spec.rb:103]: Curious if this is significant in the PR; not a blocker

File: spec/models/vita_partner_state_spec.rb [hunk 2]
```diff
@@ -46,6 +66,16 @@
...
       end
     end
+
+    context "record with duplicate state and vita partner information" do
+      let!(:vps) { create :vita_partner_state, state: "OK" }
+
+      it "is not valid" do
+        duplicate_vps = described_class.new(state: "OK", vita_partner: vps.vita_partner)
+
+        expect(duplicate_vps).not_to be_valid
```
Comment (alaroia-cfa) [inline spec/models/vita_partner_state_spec.rb:76]: Yippee!

**Review by alaroia-cfa — APPROVED** [review]
Splendid! I love validation. One question that's non-urgent and not a blocker.

File: spec/features/web_intake/routing_spec.rb (hunk 1 above)
Comment (embarnard) [reply spec/features/web_intake/routing_spec.rb:103]: @alaroia-cfa Oh yeah I think that this is just to confirm it is not routing by source param

File: spec/features/web_intake/routing_spec.rb [hunk 3]
```diff
@@ -84,6 +86,39 @@
...
+    check "None of the above"
+    click_on "Continue"
+
+    expect(page).to have_text "Just a few simple steps to file!"
+    click_on "Continue"
+
+    expect(page).to have_text "let's get some basic information"
+    fill_in "Preferred name", with: "Luna Lovegood"
+    fill_in "ZIP code", with: "28806"
+    click_on "Continue"
```
Comment (alaroia-cfa) [inline spec/features/web_intake/routing_spec.rb:120]: If at this point you want to check the routing params, that might make sense to assert it's routing by state.

**Comment by bytheway875** [comment]
@embarnard Make sure to add a translation for the state routing method so that it's viewable in the hub on the client page!

**Pull request merged**